docs/upgrading-client.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]>
++++
+title = "Upgrading notary client"
+description = "How to upgrade the notary client"
+keywords = ["docker, notary, notary-client"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Upgrading the client
+
+### From `< v0.2` to `v0.2`
+
+Please download the v0.2 binary from [the Notary releases page](https://github.com/docker/notary/releases).
+
+##### New features:
+
+- Support for adding, editing, removing, and viewing information about [delegation roles](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt#L252), and adding targets to delegation roles (as opposed to the targets role, which is currently the default).
+
+	This will allow repository owners to add one or more delegated users as collaborators, allowing them to use their own keys to publish.  Previous workflow, where a repository owner had share the targets key with a collaborator in order to enable them to publish.
+
+- Support for rotating the snapshot key to one managed by the Notary server.  This is necessary in order to facility delegations without forcing users to share the snapshot key with collaborators.
+
+	A collaborator with their own user key may sign a delegation, but without server-snapshot signing, they would be unable to publish unless they also had the snapshot key.
+
+	If users want to reclaim control of their snapshot key, they can rotate the key again to a locally-generated key.
+
+- Support for remote key rotation (forcing server-managed keys to rotate).  Forcing the server to rotate a server-managed key should be a very rare occurrence, necessarily only in the case of server compromise or remote key algorithm obsolescence.
+
+- Drop-in support for user keys - users can just place their private key to be used for delegations in `~/.notary/private/tuf_keys` as `<keyID>.key`.  The PEM header "role" should be "user", although any invalid role is currently allowed.
+
+##### Incompatible changes:
+
+- Due to a bug previous versions of notary, if delegation keys are added to a repository, old clients (including, for instance, Docker versions < 1.10, which bundle older versions of Notary client) will not be able to view the repository.  Any attempt to update with old client will result in an error.
+
+	However, repositories created/updated by Notary Client v0.2 without delegations will be perfectly readable, updatable, and publishable by old clients (with the caveat of server snapshot-signing, see below).
+
+- If server-snapshot-signing is enabled for a repository, old clients will be able to read from and update the repository, but they may not publish.
```

Comment:
the way `update` is used here makes it seem like clients can somehow write to the repo.  Maybe we could remove the `update` bit, or change the old client part to: `old clients will be able to read from updated versions of repository`